plaso/parsers/text_parser.py

```diff
@@ -612,6 +612,10 @@ def ParseFileObject(self, parser_mediator, file_object, **kwargs):
         self._text_reader.SkipAhead(file_object, end)
 
       else:
+        # TODO: somewhere above here we seem to be choking on ascii-encoded
```

Comment:
Remove comment. The issue you're seeing is that the end of string token was matched before the actual end of multi-line log entry.

Reply:
Done.